app/views/practice_resources/download_and_redirect.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="grid-container">
+  <div class="grid-row">
+    <div class="grid-col-8 grid-offset-2">
+      <div class="usa-prose text-center usa-section">
+        <% if session[:download_info].present? %>
+          <p>Your download should begin shortly. It will either open in a new tab or appear in your browser's downloads folder.</p> 
+          <p>If the download does not start or open in a new tab, please ensure your popup blocker is disabled for this site and
+          <a href="<%= session[:download_info]['download_url'] %>" target="_blank" rel="noopener noreferrer">click here</a> to try downloading again.</p>
+          <p><a href="<%= session[:download_info]['practice_path'] %>">Click here</a> to visit <%= session[:download_info]['practice_name'] %> the page where the original download link can be found.</p>
```

Comment:
In general, we want to move away from using "click here" as link text since they're not as accessible. 
```suggestion
          <a href="<%= session[:download_info]['download_url'] %>" target="_blank" rel="noopener noreferrer">try downloading again.</a></p>
          <p><a href="<%= session[:download_info]['practice_path'] %>">Visit the <%= session[:download_info]['practice_name'] %> page</a> where the original download link can be found.</p>
```